chia/consensus/blockchain.py

```diff
@@ -436,12 +450,29 @@ async def _reconsider_peak(
                             key = hint
                             if key not in hint_coin_state:
                                 hint_coin_state[key] = {}
-                            hint_coin_state[key][coin_id] = lastest_coin_state[coin_id]
+                            coin_record = lastest_coin_state[coin_id]
```

Comment:
I'm skeptical of adding any more code to the `reconsider_peak` method. In fact. all of this code for subscriptions should be called in peak_post_processing, *after* receive_block returns, otherwise it becomes critical for consensus